lib/cisco_node_utils/cmd_ref/encapsulation.yaml

```diff
@@ -0,0 +1,22 @@
+# encapsulation profile
```

Comment:
1. Just wondering if this should be more narrowly defined as `encapsulation_profile` or `encapsulation_profile_vni` since it's very specific to that use?